modules/signing_scriptworker/templates/passwords-appsv.json.erb

```diff
@@ -0,0 +1,5 @@
+{
+    "project:mozilla:application-services:releng:signing:cert:release-signing": [
```

Comment:
TODO: do we want different credentials here for Autograph?

Reply:
Looks like this is the same key that's used by `android-components`, right? I think that re-using this key for other "mozilla library" signing should be fine.
Thinking about key management: downstream projects need to trust the signing key of the signed dependencies they use. It makes sense to have to manage different keys for different organizations, but if they need to add a new trusted signing key for each new _dependency_ (or group of dependencies) that they import, I'm not sure that that makes sense. Trusting a single "mozilla library" key makes the most sense, IMHO

Reply:
Sounds good, let's keep this as is 👍